ui/src/main.rs

```diff
@@ -191,6 +191,10 @@ fn copy_assets() {
 }
 
 fn main() {
+    // Attempts to increase the file desc limit on unix-like systems
+    // Note: Will be changed out in the future 
```

Comment:
Do you mind changing this to a `// TODO:` explaining why we should change it out? Just so others can know in the future. Looks good otherwise!

Reply:
Its more of a note than a todo because it may stay in. Changing it to a `TODO` would mean it will while there are still other factors to take into consideration before the approach i may do